### PR TITLE
Add single responsibility principle example

### PR DESCRIPTION
**Pull request by FoxLviv**

### CODE REVIEW

File: ConsoleApp2/ConsoleApp2/SOLID/SingleResponsibilityPrinciple.cs [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    }
+    class Phone
+    {
+        public string Model { get; set; }
+        public int Price { get; set; }
+    }
+
+    class MobileStore
+    {
+        List<Phone> phones = new List<Phone>();
```
Comment (mikhailgrushko) [inline ConsoleApp2/ConsoleApp2/SOLID/SingleResponsibilityPrinciple.cs]: please name private fields with underscore at the beginning (_)

File: ConsoleApp2/ConsoleApp2/SOLID/SingleResponsibilityPrinciple.cs [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+        public void Process()
+        {
+            string[] data = Reader.GetInputData();
+            var phone = Binder.CreatePhone(data);
+            if (Validator.IsValid(phone))
+            {
+                phones.Add(phone);
+                Saver.Save(phone, "store.txt");
+                Console.WriteLine("Данные успешно обработаны");
```
Comment (mikhailgrushko) [inline ConsoleApp2/ConsoleApp2/SOLID/SingleResponsibilityPrinciple.cs]: also you can use here Printer from another PR

File: ConsoleApp2/ConsoleApp2/SOLID/SingleResponsibilityPrinciple.cs [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+    }
+    class ConsolePhoneReader : IPhoneReader
+    {
+        public string[] GetInputData()
+        {
+            Console.WriteLine("Введите модель:");
+            string model = Console.ReadLine();
+            Console.WriteLine("Введите цену:");
+            string price = Console.ReadLine();
+            return new string[] { model, price };
```
Comment (mikhailgrushko) [inline ConsoleApp2/ConsoleApp2/SOLID/SingleResponsibilityPrinciple.cs]: better to use Model for model & price fields

File: ConsoleApp2/ConsoleApp2/SOLID/SingleResponsibilityPrinciple.cs [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+            if (data.Length >= 2)
+            {
+                int price = 0;
+                if (Int32.TryParse(data[1], out price))
+                {
+                    return new Phone { Model = data[0], Price = price };
+                }
+                else
+                {
+                    throw new Exception("Ошибка привязчика модели Phone. Некорректные данные для свойства Price");
```
Comment (mikhailgrushko) [inline ConsoleApp2/ConsoleApp2/SOLID/SingleResponsibilityPrinciple.cs]: please write comments in english

Comment (FoxLviv) [reply ConsoleApp2/ConsoleApp2/SOLID/SingleResponsibilityPrinciple.cs]: done

File: ConsoleApp2/ConsoleApp2/SOLID/SingleResponsibilityPrinciple.cs (hunk 3 above)
Comment (FoxLviv) [reply ConsoleApp2/ConsoleApp2/SOLID/SingleResponsibilityPrinciple.cs]: done

File: ConsoleApp2/ConsoleApp2/SOLID/SingleResponsibilityPrinciple.cs (hunk 2 above)
Comment (FoxLviv) [reply ConsoleApp2/ConsoleApp2/SOLID/SingleResponsibilityPrinciple.cs]: noted

File: ConsoleApp2/ConsoleApp2/SOLID/SingleResponsibilityPrinciple.cs (hunk 1 above)
Comment (FoxLviv) [reply ConsoleApp2/ConsoleApp2/SOLID/SingleResponsibilityPrinciple.cs]: done

**Pull request merged**